TradingService/TradingService.Domain/Services/MarketDataClient.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text.Json;
+using TradingService.Domain.Interfaces;
+
+namespace TradingService.Domain.Managers;
+
+public class MarketDataClient(HttpClient httpClient): IMarketDataClient
+{
+    public async Task<decimal> GetCurrentStockPriceAsync(string ticker)
+    {
+        var response = await httpClient.GetAsync($"https://localhost:7165/api/marketData/price/{ticker}");
```

Comment:
I think it's better to store first part of the URL (https://localhost:7165) in the appsettings.development file, because after deployment this way can lead to an error